task/sanity-inspect-image/0.1/sanity-inspect-image.yaml

```diff
@@ -74,9 +74,10 @@ spec:
       skopeo inspect --no-tags --raw docker://"${IMAGE_URL}" > $RAW_IMAGE_INSPECT 2>> stderr.txt || true
 
       if [ ! -z $(cat stderr.txt) ]; then
-        echo "skopeo inspect fails, the sanity-inspect-image test meets the following error:"
+        echo "skopeo inspect fails, sanity-inspect-image test meets following error:"
         cat stderr.txt
-        HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -t 'skopeo inspect meets errors')"
+        note="Task \`$(context.task.name)\` failed: Encountered error(s) while inspecting image. For details, check Tekton task log."
```

Comment:
--> note="Task \`$(context.task.name)\` failed: Encountered errors while inspecting image. For details, check Tekton task log."

(no "(s)" allowed per our style guides)